actions-services.yml

```diff
@@ -25,10 +25,9 @@ services:
       # Utilize custom metadata
       - ./development/idp-local/metadata/saml20-idp-hosted.php:/data/vendor/simplesamlphp/simplesamlphp/metadata/saml20-idp-hosted.php
       - ./development/idp-local/metadata/saml20-sp-remote.php:/data/vendor/simplesamlphp/simplesamlphp/metadata/saml20-sp-remote.php
-    
+
     env_file:
```

Comment:
I'm surprised this is fine.

Reply:
It's because `common.env` is a file committed to the repo, unlike `.env` which is in the `.gitignore`.